src/packages/version-check/src/semver.ts

```diff
@@ -0,0 +1,13 @@
+import { default as semverDiff } from "semver/functions/diff";
+import { default as semverValid } from "semver/functions/valid";
+import { default as semverGte } from "semver/functions/gte";
+
+export function isValidSemver(semver: string) {
+  return semverValid(semver);
+}
+
+export function semverUpgradeType(a: string, b: string) {
+  if (!a || !b || semverGte(a, b)) return null;
```

Comment:
I think I originally checked for a valid semver at the places where a semver could be set but you said it was unnecessary because it would only be coming from us and we could trust those sources and should remove/clean them up.

Reply:
:+1: